tests/external_repository_registry_test.py

```diff
@@ -0,0 +1,143 @@
+import os
+import subprocess
+import sys
+from pathlib import Path
+import importlib.util
+import shutil
+import pytest
+
+REPO_URL = "https://github.com/ThrudPrimrose/LayoutTransformations.git"
```

Comment:
please no personal repos here. Use something in SPCL or in github.com/dace-framework

Reply:
I added a testing repository in SPCL:
```
REPO_URL = "https://github.com/spcl/TransformationsTest.git"
```

is this OK?